src/style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
+
+body {
+  background-color: rgb(200, 215, 255);
+  font-family: "Quicksand", sans-serif;
+  color: rgb(4, 32, 68);
+}
+
+.container {
+  max-width: 65%;
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  margin: 60px auto;
+  padding: 20px;
+}
+
+.container h1 {
+  font-size: 2em;
+  font-weight: bolder;
+  text-align: center;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+}
+
+.form .head {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+form i {
+  font-size: 1.35em;
+  padding-top: 8px;
+  cursor: pointer;
+  margin-right: -10px;
+}
+
+.head i {
+  padding: 16px 0;
+}
+
+.form ul {
+  margin: 0;
+  padding: 20px 0;
+}
+
+.lists li {
+  display: flex;
+  flex-direction: row;
+  height: 50px;
+  justify-content: space-between;
+}
+
+form {
+  width: 100%;
+  display: flex;
+  gap: 20px;
+}
+
+form .add {
+  background: none;
+  border: none;
+}
+
+form i:hover {
+  transition: 0.3s ease;
+  color: #184e9e;
+}
+
+form .task-input {
+  width: 100%;
+  padding: 10px 10px;
+  border: none;
+  border-bottom: 1px solid rgb(104, 104, 104);
+}
+
+form .task-input::placeholder {
+  color: rgb(102, 102, 102);
+  font-size: 1.2em;
+  font-family: "Quicksand", sans-serif;
+  font-style: italic;
+  opacity: 0.7;
+}
+
+form .task-input:focus {
+  outline: none;
+  font-size: 1em;
+  font-family: "Quicksand", sans-serif;
```

Comment:
Great job so far. 👍  Just a slight change -  My suggestion is you may avoid font-family repeats - for better CSS best practices. You could create a root with the property font family.
and make your code look good. 💯 😄

Reply:
Thank you so much, I'll do it accordingly.